pkg/kubeletconfig/consts.go

```diff
@@ -1,5 +1,12 @@
 package kubeletconfig
 
 const (
-	MinPodPidsLimit = 4096
+	MinPodPidsLimit         = 4096
+	PodPidsLimitOption      = "pod-pids-limit"
+	PodPidsLimitOptionUsage = "Sets the requested pod_pids_limit for your custom KubeletConfig." +
+		" Must be an integer in the range 4096 - 16,384."
+	PodPidsLimitOptionDefaultValue = -1
+
+	InterativePodPidsLimitPrompt = "Pod Pids Limit?"
```

Comment:
```suggestion
	InteractivePodPidsLimitPrompt = "Pod Pids Limit?"
```

Reply:
@gdbranco You have the eyes of a 🥷 . Nice spot! Fixed.